src/main/java/ru/education_platform_jma/jmacourseservice/controller/CourseController.java

```diff
@@ -0,0 +1,112 @@
+package ru.education_platform_jma.jmacourseservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ru.education_platform_jma.jmacourseservice.model.Course;
+import ru.education_platform_jma.jmacourseservice.repository.CourseRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+
+@RestController
```

Comment:
Use special annotations for swagger documentation (@Operation,  @Parameter, @ApiResponses, @Api etc).